templates/consortial_billing/manager.html

```diff
@@ -0,0 +1,301 @@
+{% extends "admin/core/base.html" %}
+{% load admin_urls %}
+
+{% block title %}Consortial Billing - {{ request.press.name }}{% endblock title %}
+
+{% block breadcrumbs %}
+{{ block.super }}
+<li><a href="{% url 'supporters_manager' %}">Supporters Manager</a></li>
+{% endblock %}
+
+
+{% block body %}
+{% if request.user.is_staff %}
+    <div class="row">
+        <div class="large-6 columns">
+            <div class="box">
+                <div class="title-area">
+                    <h2>Data</h2>
+                </div>
+                <div class="content">
+                    <p>
+                        You can view supporter records and other data here. Note that bands are created
+                        by the plugin when needed, so you don't have to create or edit them one by one.
+                        You can control them using Configuration below.
+                    </p>
+                    <div class="stacked button-group">
+                        <a
+                            href="/admin/consortial_billing/supporter/"
```

Comment:
It is possible to reverse admin urls (https://docs.djangoproject.com/en/dev/ref/contrib/admin/#reversing-admin-urls)